src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -404,7 +404,7 @@ def validate_addons(namespace):
             matches = str(matches)[1:-1]
             all_addons = list(ADDONS)
             all_addons = str(all_addons)[1:-1]
-            if len(matches) == 0:
+            if not matches:
```

Comment:
reason for change?

Reply:
I got lint errors on this running python linters as specified by CLI. (len-as-condition, pylint C1801)

seems like that's a bit debatable as far as being preferable itself, though: https://github.com/PyCQA/pylint/issues/1405#issuecomment-297470257

Reply:
surprised CI didn't catch that